setup.py

```diff
@@ -18,6 +18,7 @@
         'six',
         'python-magic',
         'python-magic-bin;platform_system=="Windows"',
+        'six',
```

Comment:
Any reason to list this twice?

Reply:
I can't imagine there is, removed in my branch